node/src/Pos/Explorer/Txp/Local.hs

```diff
@@ -164,7 +164,7 @@ eTxProcessTransaction itw@(txId, TxAux {taTx = UnsafeTx {..}}) = do
                 action = eProcessTx curEpoch tx (TxExtra Nothing curTime txUndo)
                 -- NE.fromList is safe here, because if `resolved` is empty, `processTx`
                 -- wouldn't save extra value, thus wouldn't reduce it to NF
```

Comment:
Previously we didn't rely on it that much. Now it's possible that there won't be any resolved input, but tx will be considered valid. But it's only possible if `verifyAllIsKnown` which is not the case for local tx processing. If someone changes it, this code can be broken.